envision/web/src/components/vehicles.js

```diff
@@ -187,31 +202,48 @@ export default function Vehicles({
         rootMesh.addChild(instancedSubMesh);
       }
 
-      // Render bounding box for social vehicle
-      if (state.actor_type == ActorTypes.SOCIAL_VEHICLE) {
-        let boundingInfo = vehicleMeshTemplates[filename].getBoundingInfo();
-        let boxSize = boundingInfo.boundingBox.extendSize.scale(2);
-        let box = MeshBuilder.CreateBox(
-          `boundingbox-${meshId}`,
-          {
-            height: boxSize.y + 0.1,
-            width: boxSize.x + 0.1,
-            depth: boxSize.z + 0.1,
-          },
-          scene
-        );
-        box.position = boundingInfo.boundingBox.center;
-
-        let boxMaterial = new StandardMaterial(
-          `boundingbox-${meshId}-material`,
-          scene
-        );
-        boxMaterial.diffuseColor = new Color4(...color);
-        boxMaterial.specularColor = new Color3(0, 0, 0);
-        boxMaterial.alpha = 0.75;
-        box.material = boxMaterial;
-        rootMesh.addChild(box);
-      }
+      let boundingInfo = vehicleMeshTemplates[filename].getBoundingInfo();
+      let boxSize = boundingInfo.boundingBox.extendSize.scale(2);
+      let box = MeshBuilder.CreateBox(
+        `boundingbox-${meshId}`,
+        {
+          height: boxSize.y + 0.1,
+          width: boxSize.x + 0.1,
+          depth: boxSize.z + 0.1,
+        },
+        scene
+      );
```

Comment:
I think we may have to update this in the future to show the actual bounding box of the vehicle. Right now the bounding box is calculated for the vehicle mesh.

Reply:
To be clear, the bounding box in sumo and SMARTS does not match the bounding box shown in envision at the moment.

This can cause the scenario to end in envision without an apparent collision.